src/data/equiv/basic.lean

```diff
@@ -751,6 +751,55 @@ def sum_compl {α : Type*} (p : α → Prop) [decidable_pred p] :
   (a : α) (h : ¬ p a) :
   (sum_compl p).symm a = sum.inr ⟨a, h⟩ := dif_neg h
 
+open equiv
+
+variables {ε : Type*} {p : ε → Prop} [decidable_pred p]
+variables (ep ep' : perm {a // p a}) (en en' : perm {a // ¬ p a})
+
+/-- Combining permutations on `ε` that permute only inside or outside the subtype
+split induced by `p : ε → Prop` constructs a permutation on `ε`. -/
+def perm.subtype_congr : equiv.perm ε :=
```

Comment:
Confusingly, we already have `equiv.subtype_congr` which is equivalent to `perm.subtype_perm`. Do you have an opinion on which name is better, and if we should move one or the other?

Reply:
I think `equiv.subtype_congr` should be renamed to `equiv.subtype_equiv` or `equiv.subtype_attach`. And/or perhaps this `equiv.perm.subtype_congr` can be renamed to `equiv.perm.subtype_detach`. Anyone with thoughts on those?

Reply:
`equiv.subtype_equiv` sounds sensible to me.

Reply:
I've clarified the docstring; I'll refactor the names in a separate PR.